server/src/main/java/io/crate/execution/dml/Indexer.java

```diff
@@ -835,8 +835,7 @@ public Collection<Reference> insertColumns(List<Reference> columns) {
     }
 
     @SuppressWarnings("unchecked")
-    public Object[] addGeneratedValues(IndexItem item) {
-        Object[] insertValues = item.insertValues();
+    public Object[] addGeneratedValues(Object[] insertValues) {
         int numExtra = (int) undeterministic.stream()
```

Comment:
Changed signature in order not to create copy of the Item on the `RawIndexer` just to call item.values().

And copy is needed otherwise, see https://github.com/crate/crate/pull/14696#discussion_r1326056268